### ISSUE TITLE
Service update returning error

### ISSUE THREAD

**Issue by jpignata** [jpignata/fargate#11]
From @nathanpeck: 

> I notice that whenever I try to vertically scale a service using the CLI I get a response like this:

```console
fargate service update locations --cpu 1024 --memory 2048
[!] Could not describe ECS task definition
InvalidParameterException: Task Definition can not be blank.
	status code: 400, request id: 7884accc-f729-11e7-ba7a-c7393d0e382f
```

**Comment by jpignata** [comment]
Addressed in[ v0.2.2](https://github.com/jpignata/fargate/releases/tag/v0.2.2). Thanks!

**Comment by nathanpeck** [comment]
Awesome that was fast! 👍
